### PR TITLE
Task 2/connecting-application-to-the-weather-api

### PR DESCRIPTION
**Pull request by petyaNikleva**
- Connected application to the weather api
- Implemented error handling
successfully-connected-to-api
Error-handling-modal

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,14 +70,13 @@
...
     "mini-css-extract-plugin": "0.8.0",
-    "node-sass": "4.14.1",
+    "node-sass": "^4.14.1",
     "npm-check-updates": "3.1.24",
     "optimize-css-assets-webpack-plugin": "5.0.3",
     "prettier": "2.2.1",
     "pretty-quick": "1.11.1",
     "react-hot-loader": "4.13.0",
     "redux-immutable-state-invariant": "2.1.0",
-    "sass-loader": "8.0.0",
```
Comment (virzhinia-petkova-redshelf) [inline package.json:80]: what are those changes needed for?

File: src/components/constants.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
-export const MAIN_TITLE_LABEL = 'Show Me The Weather';
+export const baseUrl = 'https://pfa.foreca.com/api/v1';
+export const myToken =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9wZmEuZm9yZWNhLmNvbVwvYXV0aG9yaXplXC90b2tlbiIsImlhdCI6MTY2MzU3NjUyNywiZXhwIjo5OTk5OTk5OTk5LCJuYmYiOjE2NjM1NzY1MjcsImp0aSI6ImE5ZTliM2JkNzE4YWQ1YzAiLCJzdWIiOiJwZXR5YW5pa2xldmEiLCJmbXQiOiJYRGNPaGpDNDArQUxqbFlUdGpiT2lBPT0ifQ.zGvoW0xV9SQI6Q02eB9sUgUB-3Xnq9C_kPaLI-czLWQ';
```
Comment (virzhinia-petkova-redshelf) [inline src/components/constants.js]: lets make sure we're making the request for the token in the app

File: package.json (hunk 1 above)
Comment (petyaNikleva) [reply package.json:80]: I made some necessary configuration changes in order to work with node js version 16. Also I think that I will not use 'sass' for this app. Hope it is acceptable.

File: src/components/constants.js (hunk 2 above)
Comment (petyaNikleva) [reply src/components/constants.js]: changes are addressed @virzhinia-petkova-redshelf

File: src/server/.env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+USER = petyanikleva
+PASSWORD = NQ74bLvDNCZP
+
+PORT = 3030
```
Comment (virzhinia-petkova-redshelf) [inline src/server/.env]: good job on using an .env file. :)
But you're not supposed to commit it. I shouldn't be able to see your credentials. You can add .env to your gitignore file

Comment (petyaNikleva) [reply src/server/.env]: Thanks. It has been excluded. @virzhinia-petkova-redshelf

**Review by virzhinia-petkova-redshelf — APPROVED** [review]

**Pull request merged**